### ISSUE TITLE
Controlling stem direction for individual notes

### ISSUE THREAD

**Issue by ThorsenRune** [paulrosen/abcjs#322]
Is there a way this can be achieved?

**Comment by paulrosen** [comment]
I don't know. Is there a standard way this is expressed in ABC notation? The only thing I can think of is to sprinkle `[V:a stem=up]` where you want to change it.

**Comment by ThorsenRune** [comment]
Thanks, for me it only seems to work in heading line. It would be nice to be able to tweak single notes.
some prefix like !su!/!sd! could be stem up / down for just the following note. Could be so kind to hint the code where stemdirection is controlled.

**Comment by paulrosen** [comment]
Look for `stemdir` in abc_abstract_engraver.js. For adding a decoration - I'd prefer to do something standard, but I don't know if other packages support that. I'll check.

**Comment by paulrosen** [comment]
I haven't implemented this yet, but they way I would do it is support the `pos` directive. Then you could write:
```
d[I:pos stem up]e[I:pos stem auto]fg||
```

**Comment by ThorsenRune** [comment]
Thanks Paul, really kind of you.
I did some hacking. For a start i decided that !su! !sd! would affect the following notes  stem direction
in the parseRegularMusicLine I added a new field to the element:  el.StemDir_RT

then in AbstractEngraver.prototype.addNoteToAbcElement 
I set stemdir
if (elem.StemDir_RT) {
			stemdir=elem.StemDir_RT; //200128	RTHack
		}
if (stemdir) dir=stemdir;

It allows tweaking of lines like
[V:C1] z C' D' E' & z C'B A | z !su! C' !su!D' !su!E' & z C'B !sd!A 

![test](https://user-images.githubusercontent.com/6916927/73652197-9ce47000-4686-11ea-8c45-0e14ff768a56.jpg)
